### PR TITLE
catalog was created

### PR DESCRIPTION
**Pull request by alla-savaryn**
- [DEMO LINK](https://alla-savaryn.github.io/layout_catalog/)
- [TEST REPORT LINK](https://alla-savaryn.github.io/layout_catalog/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,12 +1,359 @@
...
+            <a class="nav__links last" href="#">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="catalog">
+      <div class="container">
+        <div 
+          class="card card--1"
```
Comment (sTorba24) [inline src/index.html:73]: not best idea to set number for card in classes

File: src/styles/blocks/header.scss [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+  justify-content: space-between;
+  box-shadow: 1px 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  display: flex;
+}
+
+.nav {
+  &__list {
```
Comment (sTorba24) [inline src/styles/blocks/header.scss:15]: create separated file for nav BEM block

**Review by sTorba24 — APPROVED** [review]
Good work!

File: src/styles/blocks/header.scss (hunk 2 above)
Comment (VitaliyBondarenko1982) [reply src/styles/blocks/header.scss:15]: looks like this comment not fixed
https://github.com/mate-academy/layout_catalog/pull/3124/files#r1191160534 or your changes not pushed

File: src/index.html (hunk 1 above)
Comment (VitaliyBondarenko1982) [reply src/index.html:73]: https://github.com/mate-academy/layout_catalog/pull/3124/files#r1191157956
fix this comment

```suggestion
          class="card container__card"
```
looks like you don't use in styles this modifier. instead create element for container and move margins from .card block to .carc__container element (BEM)

File: src/styles/blocks/card.scss [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+  padding-top: 50px;
+  margin: 0 auto;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  padding: 32px 16px 16px;
+  margin: 0 24px 46px;
```
Comment (VitaliyBondarenko1982) [inline src/styles/blocks/card.scss:15]: follow BEM rules - move margins from block to element

**Review by VitaliyBondarenko1982 — CHANGES_REQUESTED** [review]
Good job.
Please check new and fix preve comments

**Pull request closed without merging**